containers/codespaces-linux/.devcontainer/base.Dockerfile

```diff
@@ -27,7 +27,7 @@ ENV SHELL=/bin/bash \
     PIPX_HOME="/usr/local/py-utils" \
     PIPX_BIN_DIR="/usr/local/py-utils/bin" \
     RVM_PATH="/usr/local/rvm" \
-    RAILS_DEVELOPMENT_HOSTS=".githubpreview.dev" \ 
+    RAILS_DEVELOPMENT_HOSTS=".githubpreview.dev,.app.github.dev" \ 
```

Comment:
Added to `universal` image in devcontainers/images https://github.com/devcontainers/images/pull/112

Reply:
We no longer build images from this repo, so this change would not have any effect. It will be nice if we could revert changes in this file 😄

Reply:
Done!